### ISSUE TITLE
Configure Redisson to connect to Azure redis cache

### ISSUE THREAD

**Issue by MurthyKVN** [redisson/redisson#1197]
Hi,
I am using redisson 3.5.6 to connect to Azure redis cache (standard tier).  I am using Config.useSingleServer and have provided password in the redisson.yaml configuration.

It is throwing Exception: Unable to connect to Redis server

Please help me in resolving the issue

**Comment by jackygurui** [comment]
Sounds like a network/firewall issue. Can you confirm you are able to connect to the redis server via command line in the same node as the redisson?

**Comment by MurthyKVN** [comment]
@jackygurui  Thanks you for quick response.

I am able to connect to the redis instance by using jedis library.  We want to use redisson to implement hibernate cache.

And one more thing i forgot to mention is only SSL communication is enabled and the same is working fine with jedis.

**Comment by MurthyKVN** [comment]
Hi @jackygurui,

I have checked from command-line tool and it is throwing error saying "connection forcibly closed by the server".

After enabling the non-ssl port on redis instance, the issue has been resolved and I am able to run the commands successfully. I have updated the port number in the application redisson configuration and the application is up and running. 

In order to use SSL from redisson, is it mandatory to configure keystore and password? or will redisson bypass (for testing) certificate validation by default.
